docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -23,23 +23,60 @@ If an element exists on the UI only for decorative purposes it is recommended th
 
 ---
 
-## Time-based Media
+## Time-based Media (WCAG 1.2)
 
 Provide alternatives for time-based media.
 
-*This guideline covers point 1.2 Time-Based Media - Level A of the WCAG standard.*
+### Captions support for prerecorded media (WCAG 1.2.1 and 1.2.2 - Level A)
+
+This guideline addresses [1.2.1 Audio-only and Video-only (Prerecorded) - Level A](https://www.w3.org/WAI/WCAG22/quickref/#audio-only-and-video-only-prerecorded) and [1.2.2 Captions (Prerecorded) - Level A](https://www.w3.org/WAI/WCAG22/quickref/#captions-prerecorded) from the WCAG standard.
+
+Some users have disabilities that prevent them from hearing media content in an application, such as prerecorded audio or video. To ensure this content is accessible to everyone, techniques like adding captions must be incorporated. This ensures all users, regardless of hearing ability, can fully engage with the media.
 
 :white_check_mark: **Success criteria**
 
-Suppose the app you are building includes media content such as video clips or audio recordings. In that case, it is suggested to provide an alternative for users with different types of accessibility needs for them to understand the material. The suggested practice would be to:
+##### Using ExoPlayer with captions
 
-- Include controls that allow users to pause and stop media, change volume and toggle subtitles (captions)
+One way to enhance accessibility is by adding captions, either open or closed. Open captions are permanently visible as they are integrated into the video, while closed captions require a media format with a player that supports them. Most modern video players support captions, and `ExoPlayer` provides built-in support for displaying them automatically.
 
-- If a video presents information that is vital to completing workflow, provide the same content in an alternative format, such as a transcript
+To view captions in media that supports them, users must enable the _Show captions_ option. This accessibility feature can be found under **Settings** > **Accessibility** > **Caption preferences**. On this screen, users can also adjust caption size and style; however, these preferences may not work with media apps that don’t support Caption Preferences.
 
-:no_entry_sign: **Failure criteria**
+##### Custom solution
+
+If a different player is used instead of `ExoPlayer`, the status of closed captions, along with the font scale and caption style, can be checked programmatically using the `CaptioningManager`. When the _Show captions_ option is enabled, captions should be turned on in the selected player.
```

Comment:
A QoL improvement would be to add the link to the class docs. https://developer.android.com/reference/android/view/accessibility/CaptioningManager